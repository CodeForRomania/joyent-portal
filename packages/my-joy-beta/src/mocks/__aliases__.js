module.exports = {
  '^joyent-ui-toolkit/dist/es/editor$': '<rootDir>/src/mocks/editor',
  '^redux-form$': '<rootDir>/src/mocks/redux-form'
};
