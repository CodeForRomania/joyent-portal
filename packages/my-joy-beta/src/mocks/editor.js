export default () => <span>joyent-maifest-editor</span>