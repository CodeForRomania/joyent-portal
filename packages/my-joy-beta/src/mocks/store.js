import React from 'react';
import { client, store } from '@root/state/store';
import { ApolloProvider } from 'react-apollo';

export default ({ children }) => (
  <ApolloProvider client={client} store={store}>
    {children}
  </ApolloProvider>
);
