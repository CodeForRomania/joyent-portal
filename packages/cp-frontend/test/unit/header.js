/**
 * @jest-environment node
 */

// import 'jest-styled-components';

import React from 'react';
import renderer from 'react-test-renderer';
import Header from '@components/navigation/header';
import { Router } from './mocks';

it('renders <Header /> without throwing', () => {
  const tree = renderer.create(<Router><Header /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
