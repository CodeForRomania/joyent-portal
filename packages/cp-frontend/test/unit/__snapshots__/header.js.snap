// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Header /> without throwing 1`] = `
.FqmtD {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 1 auto;
  -ms-flex: 0 1 auto;
  flex: 0 1 auto;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  margin-right: -0.5rem;
  margin-left: -0.5rem;
}

.bpmDrC {
  box-sizing: border-box;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  padding-right: 0.5rem;
  padding-left: 0.5rem;
}

.dPgIpY {
  box-sizing: border-box;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  padding-right: 0.5rem;
  padding-left: 0.5rem;
}

.eBVNKv {
  background-color: ;
  padding: 0.9375rem 1.125rem 0.75rem 1.125rem;
}

.ejNIBm {
  color: ;
  font-weight: 600;
  margin: 0.1875rem 0 0 0;
}

@media only screen and (min-width: 0em) {
  .bpmDrC {
    -webkit-flex-basis: 50%;
    -ms-flex-basis: 50%;
    flex-basis: 50%;
    max-width: 50%;
    display: block;
  }
}

@media only screen and (min-width: 48em) {
  .bpmDrC {
    -webkit-flex-basis: 66.66666666666667%;
    -ms-flex-basis: 66.66666666666667%;
    flex-basis: 66.66666666666667%;
    max-width: 66.66666666666667%;
    display: block;
  }
}

@media only screen and (min-width: 64em) {
  .bpmDrC {
    -webkit-flex-basis: 83.33333333333334%;
    -ms-flex-basis: 83.33333333333334%;
    flex-basis: 83.33333333333334%;
    max-width: 83.33333333333334%;
    display: block;
  }
}

@media only screen and (min-width: 0em) {
  .dPgIpY {
    -webkit-flex-basis: 25%;
    -ms-flex-basis: 25%;
    flex-basis: 25%;
    max-width: 25%;
    display: block;
  }
}

@media only screen and (min-width: 48em) {
  .dPgIpY {
    -webkit-flex-basis: 16.666666666666668%;
    -ms-flex-basis: 16.666666666666668%;
    flex-basis: 16.666666666666668%;
    max-width: 16.666666666666668%;
    display: block;
  }
}

@media only screen and (min-width: 64em) {
  .dPgIpY {
    -webkit-flex-basis: 8.333333333333334%;
    -ms-flex-basis: 8.333333333333334%;
    flex-basis: 8.333333333333334%;
    max-width: 8.333333333333334%;
    display: block;
  }
}

<div
  className="sc-bxivhb eBVNKv"
>
  <div
    className="sc-bwzfXH FqmtD"
  >
    <div
      className="sc-htpNat bpmDrC"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          className="sc-bdVaJa kujeyI"
          src="test-file-mock"
        />
      </a>
    </div>
    <div
      className="sc-htpNat dPgIpY"
    >
      <p
        className="sc-ifAKCX ejNIBm p-eFQyuj gOKtOX"
      />
    </div>
    <div
      className="sc-htpNat dPgIpY"
    >
      <p
        className="sc-ifAKCX ejNIBm p-eFQyuj gOKtOX"
      />
    </div>
  </div>
</div>
`;
